scripts/updates/fix_IRS_spectra_links.py

```diff
@@ -0,0 +1,26 @@
+from scripts.ingests.utils import load_simpledb
+from astropy.table import Table
+
+
+SAVE_DB = True  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+
+#  link to live google sheet
+link = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vQG5cGkI2aHPHD4b6ZZPTU4jjQMirU_z-yhl5ElI3p6nCIufIL64crC-yFalF58OauWHxmYvEKR_isY/pub?gid=0&single=true&output=csv'
```

Comment:
Any concern with explicitly having this link in the code base? Not sure if this is a public or private document, but it's now part of the version history for anyone to access.

Reply:
I don't have any concern with this document being publicly discoverable.